sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -100,6 +100,10 @@ final class ManifestMetadataReader {
 
   static final String ENABLE_APP_START_PROFILING = "io.sentry.profiling.enable-app-start";
 
+  static final String REPLAYS_SESSION_SAMPLE_RATE = "io.sentry.replays.session-sample-rate";
```

Comment:
Would it make sense to have this more consistent with other options, e.g. `"io.sentry.traces.profiling.sample-rate"`
```suggestion
  static final String REPLAYS_SESSION_SAMPLE_RATE = "io.sentry.session-replay.sample-rate";
```

Reply:
hm, I actually tried to match `io.sentry.traces.sample-rate` where `traces = replays` and we have two types of sample rates for session replay. Does this make sense?

How would you name the other sampler rate in your case? `io.sentry.error-replay.sample-rate` doesn't look good tbh 😅

Reply:
yeah, js calls it e.g. `replaysOnErrorSampleRate`, but I'm good with your suggestion, and weill change it to session-replay 👍